Dockerfile

```diff
@@ -1,4 +1,11 @@
-FROM node:8-alpine
+FROM node:10-alpine
```

Comment:
Talk v4 has not been tested against NodeJS 10, so we can't safely switch to Node 10.

Reply:
Reverted to NodeJS 8